src/bootstrap/dist.rs

```diff
@@ -1303,6 +1303,11 @@ impl Step for Extended {
         let cargo_installer = builder.ensure(Cargo { stage, target });
         let rustfmt_installer = builder.ensure(Rustfmt { stage, target });
         let rls_installer = builder.ensure(Rls { stage, target });
+        let llvm_tools_installer = builder.ensure(LlvmTools {
+            stage,
+            target,
+            compiler: builder.compiler(stage, target)
```

Comment:
This should be removed -- the LlvmTools step *does not* need a compiler, it should be using the target argument instead of compiler.host.